Patches/Alpha Animals/ThingDefs_Misc/Alpha_Hediffs.xml

```diff
@@ -1,28 +1,85 @@
 <Patch>
-  <Operation Class="PatchOperationSequence">
-	<operations>
-	  <li Class="PatchOperationFindMod">
-			
-		<mods><li>Alpha Animals</li></mods>
-			
-		<match Class="PatchOperationSequence">
+	<Operation Class="PatchOperationSequence">
 		<operations>
-
-
-            <li Class="PatchOperationReplace">
-				<xpath>Defs/HediffDef[defName="AA_BurrowingUnderSand"]/stages/li/statOffsets</xpath>
-				<value>
-                                                                           <statOffsets>
-					<ArmorRating_Sharp>3</ArmorRating_Sharp>
-					<ArmorRating_Blunt>4.5</ArmorRating_Blunt>
-                  			           </statOffsets>
-				</value>
-			</li>
+			<li Class="PatchOperationFindMod">
+				<mods>
+					<li>Alpha Animals</li>
+				</mods>
+				<match Class="PatchOperationSequence">
+					<operations>
+						<li Class="PatchOperationReplace">
+							<xpath>Defs/HediffDef[defName="AA_BurrowingUnderSand"]/stages/li/statOffsets</xpath>
+							<value>
+								<statOffsets>
+									<ArmorRating_Sharp>3</ArmorRating_Sharp>
+									<ArmorRating_Blunt>4.5</ArmorRating_Blunt>
+								</statOffsets>
+							</value>
+						</li>
+						<li Class="PatchOperationReplace">
+							<xpath>Defs/HediffDef[defName="AA_AnoleGrown"]/stages/li/statOffsets</xpath>
+							<value>
+								<statOffsets>
+									<ArmorRating_Sharp>3</ArmorRating_Sharp>
+									<ArmorRating_Blunt>4.5</ArmorRating_Blunt>
+								</statOffsets>
+							</value>
+						</li>
+						<li Class="PatchOperationReplace">
+							<xpath>Defs/HediffDef[defName="AA_AnoleGrown"]/comps</xpath>
+							<value>
+								<!-- <li Class="CombatExtended.ToolCE">
+									<label>claws</label>
+									<capacities>
+										<li>Scratch</li>
+									</capacities>
+									<power>32</power>
+									<cooldownTime>2</cooldownTime>
+									<soundMeleeHit>MeleeHit_BionicPunch</soundMeleeHit>
+									<soundMeleeMiss>MeleeMiss_BionicPunch</soundMeleeMiss>
+									<armorPenetrationSharp>2</armorPenetrationSharp>
+									<armorPenetrationBlunt>3.5</armorPenetrationBlunt>
+								</li> -->
+								<comps>
 			
+									<li Class="HediffCompProperties_VerbGiver">
+										<tools>
+											<li Class="CombatExtended.ToolCE">
+												<label>claws</label>
+												<capacities>
+													<li>Scratch</li>
+												</capacities>
+												<power>32</power>
+												<cooldownTime>2</cooldownTime>
+												<soundMeleeHit>MeleeHit_BionicPunch</soundMeleeHit>
+												<soundMeleeMiss>MeleeMiss_BionicPunch</soundMeleeMiss>
+												<armorPenetrationSharp>2</armorPenetrationSharp>
+												<armorPenetrationBlunt>3.5</armorPenetrationBlunt>
+											</li>
+										</tools>
+									</li>
+									<li Class="HediffCompProperties_Immunizable">
+										<severityPerDayNotImmune>-2</severityPerDayNotImmune>
+									</li>
+									<li Class="AlphaBehavioursAndEvents.HediffCompProperties_AnoleGrown">
+				
+									</li>
```

Comment:
Lines 62-67 are redundant.

Reply:
Even if we do a patchoperationreplace? Idk how rimworld patching works in terms of replacing.

Reply:
These lines replace things with same things. Why add them and bloat the code?

Reply:
I think since this "works" (theoretical works) why change it? Because all of the comps will be replaced with the lines and wouldn't that fuck with the anole?

Reply:
This does not work. The absence of error message can mean that patch did nothing.

Reply:
Still no idea how to resolve this on the xml side of things...